cmd/minikube/cmd/completion.go

```diff
@@ -47,6 +47,16 @@ const longDescription = `Outputs minikube shell completion for the given shell (
 
 	Additionally, you may want to output the completion to a file and source in your .bashrc
 
+	Windows:
+		## Save completion code to a script and execute in the profile
+		PS> minikube completion powershell > $HOME\.minikube-completion.ps1
+		PS> Add-Content $PROFILE ". ` + "`" + `$HOME\.minikube-completion.ps1"
```

Comment:
I tried this command but it gives me a newline like the command is not complete then I enter the command
```
> Add-Content $PROFILE ". ` + "`" + `$HOME\.minikube-completion.ps1"
>>
```
I think the quote escaping might be slightly off

Reply:
You are inside Golang back-tick block. The actual output when run with `minikube completion --help` is the following:

```
        Windows:
                ## Save completion code to a script and execute in the profile
                PS> minikube completion powershell > $HOME\.minikube-completion.ps1
                PS> Add-Content $PROFILE ". `$HOME\.minikube-completion.ps1"

                ## Execute completion code in the profile
                PS> Add-Content $PROFILE "if (Get-Command minikube -ErrorAction SilentlyContinue) {
                        minikube completion powershell | Out-String | Invoke-Expression
                    }"
```

Reply:
The actual line added to your $PROFILE should be dot-sourcing of the generated file (tested right now):

```
. $HOME\.minikube-completion.ps1
```

Reply:
The same effect could be achieved by using non-interpreting single-quotes `Add-Content $PROFILE '. $HOME\.minikube-completion.ps1'`, which does not use the PowerShell back-tick escaping and is probably more readable. I will change it.

Reply:
Ah, didn't realize, thanks for the clarification